### PR TITLE
feat: Add support for forced subtitles

### PR DESCRIPTION
**Pull request by avelad**

### CODE REVIEW

File: streamer/autodetect.py [hunk 1]
```diff
@@ -173,3 +173,13 @@ def get_channel_layout(input: Input) -> Optional[AudioChannelLayoutName]:
       return bucket.get_key()
 
   return None
+
+def get_forced_subttitle(input: Input) -> bool:
+  """Returns the autodetected channel count of the input."""
```
Comment (mariocynicys) [inline streamer/autodetect.py:178]: unrelated doc comment

File: streamer/output_stream.py [hunk 2]
```diff
@@ -36,6 +36,7 @@ def __init__(self,
     self.input: Input = input
     self.features: Dict[str, str] = {}
     self.codec: Union[AudioCodec, VideoCodec, None] = codec
+    self.forced_subtitle = False;
```
Comment (mariocynicys) [inline streamer/output_stream.py:39]: isn't that already in `Input`? why repeat it again here if we could use `self.input.forced_subtitles` when we need to query the value

Comment (avelad) [reply streamer/output_stream.py:39]: I have simplified the code, and I have also followed your suggestion since this only applies to subtitles.

Comment (mariocynicys) [reply streamer/output_stream.py:39]: well, i mean that `self.input` declared 3 lines above already has this data, so we don't need to copy it further in output stream. so `forced_subtitle` field here isn't really that useful.

File: streamer/packager_node.py [hunk 3]
```diff
@@ -144,6 +144,9 @@ def _setup_stream(self, stream: OutputStream) -> str:
     if stream.input.drm_label:
       dict['drm_label'] = stream.input.drm_label
 
+    if stream.type == MediaType.TEXT and stream.forced_subtitle:
```
Comment (mariocynicys) [inline streamer/packager_node.py:147]: and instead here we could use `stream.input.forced_subtitles` to query this value.

File: streamer/output_stream.py [hunk 4]
```diff
@@ -155,6 +156,8 @@ def __init__(self,
     super().__init__(MediaType.TEXT, input, codec, pipe_dir,
                      skip_transcoding, pipe_suffix='.vtt')
 
+    self.forced_subtitle = input.forced_subtitle
+
```
Comment (mariocynicys) [inline streamer/output_stream.py:160]: and remove this as well ofc.

Comment (avelad) [reply streamer/output_stream.py]: Done!

File: streamer/packager_node.py (hunk 3 above)
Comment (avelad) [reply streamer/packager_node.py]: Done!

File: streamer/output_stream.py (hunk 2 above)
Comment (avelad) [reply streamer/output_stream.py]: Done!

**Review by mariocynicys — APPROVED** [review]
LGTM!
Thanks :)

**Pull request merged**